templates/github/.ci/scripts/release.py.j2

```diff
@@ -76,12 +76,12 @@ with open(f"{plugin_path}/setup.py") as fp:
 release_version = version.replace(".dev", "")
 
 
-issues_to_close = []
+issues_to_close = set()
 for filename in Path(f"{plugin_path}/CHANGES").rglob("*"):
     if filename.stem.isdigit():
         issue = filename.stem
         issue_url = f"{REDMINE_URL}/issues/{issue}.json"
```

Comment:
Also, I can't tell what this line is doing. I think it can be removed?

Reply:
I think so